src/script.js

```diff
@@ -0,0 +1,265 @@
+const roomURL = 'DAILY_ROOM_URL';
+const kahootQuizURL =
+  'https://play.kahoot.it/v2/?quizId=c990a754-237c-4d1a-bfe5-c580895d7f5f';
+const hideClassName = 'hidden';
+
+const Integration = {
```

Comment:
This is a bit of refactoring that I hope will make toggling different integrations (player vs host) more intuitive -  I'll point out where it is used below.